recipes/bcftools/1.10/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+set -ex
+
+n="$CPU_COUNT"
+
+CPPFLAGS="-I$PREFIX/include" LDFLAGS="-L$PREFIX/lib" \
+        ./configure --prefix="$PREFIX" --with-htslib="$PREFIX" 
```

Comment:
We now prefer to pass these to `configure` using command line parameters e.g.

`./configure [OPTION]... [VAR=VALUE].`

```
./configure --prefix="$PREFIX" --with-htslib="$PREFIX" \
  CPPFLAGS="-I$PREFIX/include" LDFLAGS="-L$PREFIX/lib"
```